deployments/liqo/templates/liqo-auth-deployment.yaml

```diff
@@ -67,6 +67,22 @@ spec:
           - "$(POD_NAMESPACE)"
           - "--resyncSeconds"
           - "30"
+          {{- if .Values.awsConfig.accessKeyId }}
+          - "--awsAccessKeyId"
+          - {{ .Values.awsConfig.accessKeyId | quote }}
```

Comment:
To me, it seems more standard (and easier to read) if key and value are on the same line